test/e2e/kuttl/inject-flagd/00-assert.yaml

```diff
@@ -1,10 +1,4 @@
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: flagd-query-test
-status:
-  conditions:
-    - type: Complete
-      status: 'True' #this ensures flagd is running
-  succeeded: 1
-
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - command: kubectl wait --for=condition=complete job flagd-query-test -n $NAMESPACE
```

Comment:
same check just like this we are sure we are looking for the namespace job and this is easily seen in the test logs e.g. 
`| running command: [kubectl wait --for=condition=complete job flagd-query-test -n kuttl-test-ultimate-martin]`